microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,106 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+* [CoreWebView2Frame.PostSharedBufferToScript Method](/dotnet/api/microsoft.web.webview2.core.corewebview2frame.postsharedbuffertoscript?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease#microsoft-web-webview2-core-corewebview2frame-postsharedbuffertoscript(microsoft-web-webview2-core-corewebview2sharedbuffer-microsoft-web-webview2-core-corewebview2sharedbufferaccess-system-string))
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=&preserve-view=truewebview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+* [CoreWebView2Frame.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Experimental18::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimental18?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+* [ICoreWebView2ExperimentalFrame4::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe4?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+
+---
+
+*  Added support by adding these properties to run JavaScript code from the JavaScript parameter in the current top-level document:
+    *  `ColumnNumber`
+    *  `LineNumber`
+    *  `Message`
+    *  `Name`
+    *  `ToJson`
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ScriptException Class](/dotnet/api/microsoft.web.webview2.core.corewebview2scriptexception?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ScriptException Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2scriptexception?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalScriptException interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalscriptexception?view=webview2-1.0.1466-prerelease&preserve-view=true)
+
+---
+
+#### Bug fixes
+
+*   Fixed a bug which custom header title in print settings could be wrong([Issue #2093](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2093))
+*   Display AllowedCertificateAuthorities in add_ClientCertificateRequested event as Base64 string. (Runtime)([Issue #2346](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2346))

```

Comment:
```suggestion
*   Display `AllowedCertificateAuthorities` in `add_ClientCertificateRequested` event as a `Base64` string. (Runtime)([Issue #2346](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2346))
```
If we wanted more clarity, could add "instead of binary data"